sdk-setup/src/mb2

```diff
@@ -731,6 +731,9 @@ GLOBAL OPTIONS
         packages, read the passphrase from the <file>. Only the first line will
         be read.
 
+    --package-timeline
```

Comment:
What's the story behind the name? Why not '--preserve-old-packages' ?

Reply:
In order to group well, the name should begin with "package" (`--foo-bar` is often represented as `foo.bar` on sfdk side), but `--package-preserve-old` would not read well. Hence the option to get a timeline of your packaged work, which fits well between the other `package.*` options.